src/importexport/musicxml/tests/data/testTremolo.xml

```diff
@@ -293,7 +293,7 @@
           <actual-notes>2</actual-notes>
           <normal-notes>1</normal-notes>
           </time-modification>
-        <stem>down</stem>
+        <stem>up</stem>
```

Comment:
It would be great if you could add new tests that would guard your fix and protect us from regressions

Reply:
Apologies for the delay, I have been away for a few days.

This utest change was a dumb mistake by me; after checking `testTwoNoteTremoloTuplet.xml`, which contained what now amounts to an illegal two note tremolo (a tremolo consisting of two notes with opposite stem directions) I thought I had checked both failing tests and that they both had the same problem. They did not, so all changes to `testTremolo.xml` have been reverted.

There is still a change to `testTwoNoteTremoloTuplet.xml`, because it contains this:
![image](https://user-images.githubusercontent.com/89263931/173395726-3d54ccfe-6435-4ea3-8eb3-d0cc26343976.png)
...which is not a valid tremolo anymore. And I might also add that if this xml tests tremolo tuplet and fails on tremolos that are not tuplets, it's a very strangely designed test.

I will add a utest that deals with flipping one note of a two-note tremolo ensuring that the other is also flipped. This could take me some time as I'm not 100% familiar with testing of commands.

Reply:
I guess now thinking about it, the test itself might have to be modified anyway to A) make more sense and test the things it's supposed to test and nothing more, or at least B) have a reference file to show the differences between the musicxml and the musescore treatment, which will be different from the imported data (with explicit stem directions). What do you think?